src/helpers/avatarGenerator.js

```diff
@@ -0,0 +1,15 @@
+const jdenticon = require("jdenticon");
+const path = require("path");
+const fs = require("fs");
+
+async function avatarGenerator(email, size) {
+  const avatarPath = path.join(process.cwd(), "src", "public", "images");
+  const avatarName = Date.now() + ".png";
+
+  const avatar = jdenticon.toPng(email, size);
+  await fs.promises.writeFile(avatarPath + "/" + avatarName, avatar);
```

Comment:
do not use concatenation with `sting path`

you should use `path.join` to join `sting path`